config/routes.rb

```diff
@@ -1,7 +1,7 @@
 Rails.application.routes.draw do
   resources :agencies
   devise_for :users, controllers: { registrations: "registrations" }
-  root 'home#index'
+  root 'reports#index'
```

Comment:
We can do, although eventually we will need a home page to direct people to if they're not logged in